src/lib.rs

```diff
@@ -208,6 +229,16 @@
 //! [`MaxLevelFilter`]: struct.MaxLevelFilter.html
 //! [`set_logger_raw`]: fn.set_logger_raw.html
 //! [`shutdown_logger_raw`]: fn.shutdown_logger_raw.html
+//! [env_logger]: https://docs.rs/env_logger/*/env_logger/
+//! [simple_logger]: https://docs.rs/simple_logger/*/simple_logger/
```

Comment:
https://github.com/borntyping/rust-simple_logger would be a better landing page.